coordinate/client.go

```diff
@@ -205,6 +207,11 @@ func (c *Client) Update(node string, other *Coordinate, rtt time.Duration) (*Coo
 		return nil, err
 	}
 
+	durSec := rtt.Seconds()
```

Comment:
You can do this as a `time.Duration` (I'd put the constant right here, too):

```
const maxRTT = 10 * time.Second
if rtt < 0 || rtt > maxRTT {
    return nil, fmt.Errorf("RTT %s out of range 0 to %s", rtt, maxRTT)
}
```

Reply:
I'd also rail it way lower since 10 seconds is already several times around the Earth, so this should keep the coordinates better behaved in the face of weirdness (and it's not useful to model anything slower).